perllib/FixMyStreet/App/Controller/Admin.pm

```diff
@@ -824,6 +824,82 @@ sub report_edit : Path('report_edit') : Args(1) {
     return 1;
 }
 
+sub templates : Path('templates') : Args(0) {
+    my ( $self, $c ) = @_;
+
+    $c->detach( '/page_error_404_not_found' )
+        unless $c->cobrand->moniker eq 'zurich';
+
+    my $user = $c->user;
+
+    $self->templates_for_body($c, $user->from_body );
+}
+
+sub templates_view : Path('templates') : Args(1) {
+    my ($self, $c, $body_id) = @_;
+
+    # e.g. for admin
+
+    my $body = $c->model('DB::Body')->find($body_id)
+        or $c->detach( '/page_error_404_not_found' );
+
+    $self->templates_for_body($c, $body);
+}
+
+sub template_edit : Path('templates') : Args(2) {
```

Comment:
Might be worth putting the `unless $c->cobrand->moniker eq 'zurich'` check on this and `templates_view` too, given that the HTML templates don't exist anywhere except the Zürich cobrand.